test/test.js

```diff
@@ -323,6 +323,63 @@ describe('csurf', function () {
     })
   })
 
+  describe.only('req.verifyToken()', function () {
+    var app, token, cookie;
+    beforeEach(function(done) {
```

Comment:
No, you don't technically have to, but I personally prefer to, so side effects from one test don't effect the other tests. I'll make it before to make it consistent though.